syntax = "proto3";

service lookup {
	rpc lookup(lookupmsg) returns (lookupReply) {}
}

message lookupmsg {
	string cmd = 1;
	string value = 2;
	string user = 3;
	string channel = 4;
}

message lookupReply {
	string response = 1;
	string user = 2;
	string channel = 3;
}